packages/manager/apps/zimbra/src/pages/dashboard/Domains/ModalDeleteDomain.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react';
+import { useSearchParams, useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_COLOR_HUE,
+} from '@ovhcloud/ods-common-theming';
+import { ODS_ICON_NAME } from '@ovhcloud/ods-components';
+import { OsdsMessage, OsdsText } from '@ovhcloud/ods-components/react';
+import { useNotifications } from '@ovhcloud/manager-components';
+import { useQuery } from '@tanstack/react-query';
+import { useOrganization, usePlatform } from '@/hooks';
+import { deleteZimbraPlatformDomain } from '@/api/DELETE/apiv2/services';
+import Modal from '@/components/Modals/Modal';
+import {
+  getZimbraPlatformDomainDetail,
+  getZimbraPlatformDomainsQueryKey,
+} from '@/api';
+
+export default function ModalDeleteDomain() {
+  const [searchParams] = useSearchParams();
+  const deleteDomainId = searchParams.get('deleteDomainId');
+  const { t } = useTranslation('domains/delete');
+  const { platformId } = usePlatform();
+  const { data: organization } = useOrganization();
+  const { addError, addSuccess } = useNotifications();
+  const navigate = useNavigate();
+  const onClose = () => {
+    const organizationId = organization?.id
+      ? `?organizationId=${organization.id}`
+      : '';
+    return navigate(`..${organizationId}`);
+  };
+  const { data, isLoading } = useQuery({
+    queryKey: getZimbraPlatformDomainsQueryKey(platformId, deleteDomainId),
+    queryFn: () => getZimbraPlatformDomainDetail(platformId, deleteDomainId),
+  });
+
+  const [hasEmailAccount, setHasEmailAccount] = useState(false);
+  useEffect(() => {
+    setHasEmailAccount(
+      data?.currentState.accountsStatistics[0].configuredAccountsCount > 0,
+    );
+  }, [isLoading]);
```

Comment:
I think it would be efficient if you do a call on  /zimbra/platform/{platformId}/account with domain id params